### ISSUE TITLE
Is it possible to run pluginsync for plugin available outside intelsdi-x?

### ISSUE THREAD

**Issue by mkleina** [intelsdi-x/snap-pluginsync#81]
For example, I am writing new plugin and it is not available on intelsdi-x yet, but I want to initialize `scripts` folder and `.pluginsync.yml` file directly on my disk or on my private GitHub branch, how to do it?

**Comment by nanliu** [comment]
Yes, but you'll need to do some customization. I would do the following steps:

* fork this repo
* strip out or modify files in moduleroot directory as you see necessary (LICENSE etc)
* change [modulesync.yml](https://github.com/intelsdi-x/snap-pluginsync/blob/master/modulesync.yml#L3) namespace to your user account instead of intelsdi org
* add/replace [managed_modules](https://github.com/intelsdi-x/snap-pluginsync/blob/master/managed_modules.yml) with your repo name.

HTH.
